hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleSchedule.java

```diff
@@ -74,6 +74,7 @@
     private RichInstant schedulingTXValidStart;
     private Set<JKey> signers = new LinkedHashSet<>();
     private boolean deleted;
+    private long expiry;
```

Comment:
We will have to update the `ScheduleEntitySizes` as-well more specifically the `fixedBytesInScheduleRepr`